firefox-ios/firefox-ios-tests/Tests/XCUITests/BookmarksTests.swift

```diff
@@ -9,6 +9,7 @@ let url_1 = "test-example.html"
 let url_2 = ["url": "test-mozilla-org.html", "bookmarkLabel": "Internet for people, not profit — Mozilla"]
 let urlLabelExample_3 = "Example Domain"
 let url_3 = "localhost:\(serverPort)/test-fixture/test-example.html"
+let urlBarAddress = XCUIApplication().textFields[AccessibilityIdentifiers.Browser.UrlBar.searchTextField]
```

Comment:
This is a great idea and suggestion. Can't we add this in the FxScreenGraph or the BaseTestCase, that would simplify things even more. 
Similarly as we defined some options: https://github.com/mozilla-mobile/firefox-ios/blob/f387704352915d6cd896b326ebdcaaddbf3658c4/firefox-ios/firefox-ios-tests/Tests/XCUITests/FxScreenGraph.swift#L106

Reply:
i will give it a try

Reply:
Moved it to BaseTestCase